lib/tasks/check_content_consistency.rake

```diff
@@ -1,32 +1,31 @@
-namespace :check_content_consistency do
-  def check_content(content_id, locale, ignore_recent)
-    checker = ContentConsistencyChecker.new(content_id, locale, ignore_recent)
-    errors = checker.call
+require "content_consistency_checker"
 
-    if errors.any?
-      puts "#{content_id} #{locale} 😱"
-      puts errors
-    end
+def report_errors(errors, content_store)
+  Airbrake.notify(
+    "Inconsistent #{content_store} documents",
```

Comment:
maybe name this more explicitly - "Documents inconsistent with the #{content_store} content store" ?